scripts/qa_crowbarsetup.sh

```diff
@@ -3924,6 +3926,7 @@ function onadmin_addupdaterepo()
                 --accept x86_64.rpm,noarch.rpm \
                 ${repo%/}/
         done
+        onadmin_reporeplace
```

Comment:
because we often want to add extra repos before prepare-crowbar-install step, so that extra packages can be used there. And we need the createrepo rpm available.